subsquid/src/config.ts

```diff
@@ -27,7 +27,7 @@ export const archive = (): string => {
 
 export const firstBlock = (): number => {
   const relayChain = chain();
-  if (relayChain === "wss://rpc.composablenodes.tech") {
+  if (relayChain.toLowerCase().includes("picasso")) {
```

Comment:
I think for now this is fine, but why not set firstBlock with an environment variable?

Reply:
With a default if not provided based on this approach?

Reply:
Makes sense, done.